README.md

```diff
@@ -54,6 +61,14 @@ The data for importing is already in the exported SQL dumps in the `data/export`
 ./data/init_db.sh
 ```
 
+### Generate Embeddings for Pruning
+
+We use an [embedding-based method](https://github.com/defog-ai/sql-eval/blob/main/utils/pruning.py) to prune the metadata that goes into the prompt. Thus, you would need to download the spacy model used in our NER heuristic, and then generate the pickle files that will contain the pre-processed embeddings:
+```bash
+python -m spacy download en_core_web_sm
+python data/gen_embeddings.py
```

Comment:
This is not strictly needed because the pickle files will already be downloaded! Apart from this, looks excellent. Thank you!

Reply:
Oh ok got it! Lemme remove the gen_embeddings bit

Reply:
Lemme get them to download spacy right at the start with installing requirements.txt